server/tokens.go

```diff
@@ -259,8 +260,9 @@ func (c *Server) blindedTokenRedeemHandlerV3(w http.ResponseWriter, r *http.Requ
 			}
 
 		}
-		response = blindedTokenRedeemResponse{verifiedCohort}
+		response = blindedTokenRedeemResponse{issuer.IssuerCohort}
```

Comment:
do the cohorts actually work as expected for v3 issuers? it seems like we only ever retrieve one issuer

Reply:
removing the concept of issuers for time aware creds.

Reply:
https://github.com/brave-intl/challenge-bypass-server/issues/256